### PR TITLE
doc/md: guide for sequelize schema loader

### PR DESCRIPTION
**Pull request by ronenlu**

### CODE REVIEW

**Review by masseelch — APPROVED** [review]

File: doc/md/guides/orms/sequelize.mdx [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+
+To use Atlas with Sequelize, users can utilize the [Sequelize Atlas Provider](https://github.com/ariga/atlas-provider-sequelize),
+a small program that can be used to load the schema of a Sequelize project into Atlas.
+
+In this guide, we will show how Atlas can be used to automatically plan schema migrations for
+Sequelize users.
+
+## Prerequisites
+
+* A local [sequelize](https://github.com/sequelize/sequelize) project.
```
Comment (rotemtam) [inline doc/md/guides/orms/sequelize.mdx:47]:
```suggestion
* A local [Sequelize](https://github.com/sequelize/sequelize) project.
```

File: doc/md/guides/orms/sequelize.mdx [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+The problem with [creating migrations](https://sequelize.org/docs/v6/other-topics/migrations/#creating-the-first-model-and-migration)
+in Sequelize is that they are usually written by hand, which is error-prone and
+time-consuming.
+
+Atlas can automatically plan database schema migrations for developers using Sequelize.
+Atlas plans migrations by calculating the diff between the _current_ state of the database,
+and its _desired_ state.
+
+In the context of [versioned migrations](/concepts/declarative-vs-versioned#versioned-migrations), the current state can be thought of as the database schema that would have
+been created by applying all previous migration scripts.
```
Comment (rotemtam) [inline doc/md/guides/orms/sequelize.mdx:34]: pay attention to line length
```suggestion
In the context of [versioned migrations](/concepts/declarative-vs-versioned#versioned-migrations),
the current state can be thought of as the database schema that would have been created by applying
all previous migration scripts.
```

File: doc/md/guides/orms/sequelize.mdx [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+
+If you don't have a Sequelize project handy, you can use [sequelize/express-example](https://github.com/sequelize/express-example)
+as a starting point:
+
+```
+git clone git@github.com:sequelize/express-example.git
+```
+
+## Using the Atlas Sequelize Provider
+In this guide, we will use the [Sequelize Atlas Provider](https://github.com/ariga/atlas-provider-sequelize)
```
Comment (rotemtam) [inline doc/md/guides/orms/sequelize.mdx:57]:
```suggestion

In this guide, we will use the [Sequelize Atlas Provider](https://github.com/ariga/atlas-provider-sequelize)
```

File: doc/md/guides/orms/sequelize.mdx [hunk 4]
```diff
@@ -0,0 +1,482 @@
...
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```bash
+npm install @ariga/ts-atlas-provider-sequelize
+```
+</TabItem>
+</Tabs>
+
+Make sure all you Node dependencies are installed by running:
```
Comment (rotemtam) [inline doc/md/guides/orms/sequelize.mdx:85]:
```suggestion
Make sure all your Node dependencies are installed by running:
```

File: doc/md/guides/orms/sequelize.mdx [hunk 5]
```diff
@@ -0,0 +1,482 @@
...
+```bash
+npm install
+```
+
+### Standalone vs Script mode
+
+The Atlas Sequelize Provider can be used in two modes:
+* **Standalone** - If all of your Sequelize models exist in single directory,
+ you can use the provider directly to load your Sequelize schema into Atlas.
```
Comment (rotemtam) [inline doc/md/guides/orms/sequelize.mdx:94]: I think a directory is called a module in this context? 
```suggestion
* **Standalone** - If all of your Sequelize models exist in a single Node module,
 you can use the provider directly to load your Sequelize schema into Atlas.
```

File: doc/md/guides/orms/sequelize.mdx [hunk 6]
```diff
@@ -0,0 +1,482 @@
...
+* **Script** - In other cases, you can use the provider as a npm package to write a script
+ that loads your Sequelize schema into Atlas.
```
Comment (rotemtam) [inline doc/md/guides/orms/sequelize.mdx:96]:
```suggestion
* **Script** - In other cases, you can use the provider as an npm package to write a script
 that loads your Sequelize schema into Atlas.
```

File: doc/md/guides/orms/sequelize.mdx [hunk 7]
```diff
@@ -0,0 +1,482 @@
...
+  `createdAt` datetime NOT NULL,
+  `updatedAt` datetime NOT NULL,
+  `userID` int NOT NULL,
+  PRIMARY KEY (`id`),
+  INDEX `userID` (`userID`),
+  CONSTRAINT `Tasks_ibfk_1` FOREIGN KEY (`userID`) REFERENCES `Users` (`id`) ON UPDATE CASCADE ON DELETE NO ACTION
+) CHARSET utf8mb4 COLLATE utf8mb4_0900_ai_ci;
+```
+
+Amazing, Atlas automatically generated a migration file that will create the `Users` and `Tasks` tables in our database!
```
Comment (rotemtam) [inline doc/md/guides/orms/sequelize.mdx:429]:
```suggestion
Amazing! Atlas automatically generated a migration file that will create the `Users` and `Tasks` tables in our database!
```

**Review by rotemtam — APPROVED** [review]
Please see comments first

**Pull request merged**